exist-core/src/test/xquery/xquery3/serialize.xql

```diff
@@ -476,508 +448,447 @@ math:pi#0
+    serialize(doc($ser:collection || "/test-xsl.xml"),
+        map { "exist:process-xsl-pi": true() })
 };
 
 declare
     %test:assertXPath("contains($result, 'stylesheet')")
 function ser:exist-process-xsl-pi-false() {
-    let $doc := doc($ser:collection || "/test-xsl.xml")
-    return fn:serialize($doc, map {xs:QName("exist:process-xsl-pi"): false()})
+    serialize(doc($ser:collection || "/test-xsl.xml"),
+        map { "exist:process-xsl-pi": false() })
 };
 
 declare
+    %test:args("text")
     %test:assertEquals("1--2")
-function ser:item-separator-text-method() {
-    let $data := (1, 2)
-    return ser:serialize-with-item-separator($data, "text")
-};
-
-declare
+    %test:args("html")
     %test:assertEquals("1--2")
-function ser:item-separator-html-method() {
-    let $data := (1, 2)
-    return ser:serialize-with-item-separator($data, "html")
-};
-
-declare
+    %test:args("xhtml")
     %test:assertEquals("1--2")
-function ser:item-separator-xhtml-method() {
-    let $data := (1, 2)
-    return ser:serialize-with-item-separator($data, "xhtml")
-};
-
-declare
+    %test:args("xml")
     %test:assertEquals("1--2")
-function ser:item-separator-xml-method() {
-    let $data := (1, 2)
-    return ser:serialize-with-item-separator($data, "xml")
-};
-
-declare
+    %test:args("adaptive")
     %test:assertEquals("1--2")
-function ser:item-separator-adaptive-method() {
-    let $data := (1, 2)
-    return ser:serialize-with-item-separator($data, "adaptive")
+function ser:item-separator-with-method($method as xs:string) {
+    serialize((1, 2),
+        <output:serialization-parameters
+                xmlns:output="http://www.w3.org/2010/xslt-xquery-serialization">
+            <method>{$method}</method>
+            <item-separator>--</item-separator>
+        </output:serialization-parameters>
+    )
 };
 
 declare
     %test:assertEquals("1|2|3|4|5|6|7|8|9|10")
 function ser:serialize-xml-033() {
     let $params :=
         <output:serialization-parameters xmlns:output="http://www.w3.org/2010/xslt-xquery-serialization">
-            <output:method value="xml"/>
-            <output:item-separator value="|"/>
+            <method value="xml"/>
+            <item-separator value="|"/>
         </output:serialization-parameters>
     return serialize(1 to 10, $params)
 };
 
+declare
+    %test:assertEquals("1|2|3|4|5|6|7|8|9|10")
+function ser:serialize-xml-133() {
+    serialize(1 to 10, map { "method": "xml", "item-separator": "|" })
+};
+
 declare
     %test:assertEquals("1==2==3==4")
 function ser:serialize-xml-034() {
-    let $params :=
+    serialize(1 to 4,
         <output:serialization-parameters xmlns:output="http://www.w3.org/2010/xslt-xquery-serialization">
-            <output:method value="xml"/>
-            <output:omit-xml-declaration value="yes"/>
-            <output:item-separator value="=="/>
+            <method value="xml"/>
+            <omit-xml-declaration value="yes"/>
+            <item-separator value="=="/>
```

Comment:
The spec requires that these elements be in the `output` namespace (even if, evidently, eXist doesn't care). Please fix the namespace on the child elements.

Reply:
My understanding of namespaces in XML is that elements are put into the namespace of the parent element.

Reply:
Elements *inherit their parent's in-scope namespaces*, but *not necessarily their namespace*. See https://www.w3.org/TR/xml-names/#scoping-defaulting.

In this case the `<output:serialization-parameters>` element does create a binding between the `output` namespace prefix and the `http://www.w3.org/2010/xslt-xquery-serialization` namespace URI, but unprefixed elements will remain in the empty namespace. 

To illustrate this, give this a try:

```xquery
xquery version "3.1";

declare namespace output="http://www.w3.org/2010/xslt-xquery-serialization";

let $root := 
    <output:serialization-parameters xmlns:output="http://www.w3.org/2010/xslt-xquery-serialization">
        <method>adaptive</method>
    </output:serialization-parameters>
return
    $root/output:*
```

The only way the parent could cause its child to be in its namespace would be to add an `xmlns` namespace node without a prefix - i.e., changing the default element namespace within the scope of the parent. 

Doing so here will place the parent's children in the `output` namespace, and the query that selects child elements in the `output` namespace will now return the child.

```xquery
xquery version "3.1";

declare namespace output="http://www.w3.org/2010/xslt-xquery-serialization";

let $root := 
    <output:serialization-parameters xmlns="http://www.w3.org/2010/xslt-xquery-serialization"
                                     xmlns:output="http://www.w3.org/2010/xslt-xquery-serialization">
        <method>adaptive</method>
]    </output:serialization-parameters>
return
    $root/output:*
```

A useful function for finding out the namespace URI for a node is `fn:namespace-uri()`.

Reply:
in 5a7e87fc5954ffc84e439ffa276bda51273f1a29 I addressed your review @joewiz 

- I opted for declaring the output namespace once at the top of the testuite. 
  This way it does not have to be redeclared in each test case. 
- All children of `output:serialization` are now also put back into that namespace
  I think we should open a separate issue on the finding that the namespace of child elements is effectively ignored.

Reply:
@line-o Perfect! Thank you! And I agree, we should open a new issue about the failure to check namespace of serialization parameters.